### DISCUSSION TITLE
how can I see logs emitted during a test run?

### DISCUSSION THREAD

**Discussion by dhduvall** [renovatebot/renovate#25343] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.31.1

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'm trying to fix a stack trace I recently started seeing, and although I have a fix that works, I'm struggling a bit with the test (it's not covering the new lines and apparently doesn't even properly test the fix). I was hoping to be able to look at the logs which were emitted by `renovate` during the test run in order to help debug it, but I can't figure out how to see them.

The fix and not-working test are in https://github.com/dhduvall/renovate/tree/duvall/nuget-nopagekbaseaddress

If y'all know what I need to do to make the test work, I'm happy to just do that, but it'd be nice to know how to see the logs, too, in case I run into this in the future.

### What you tried so far.

- setting `LOG_LEVEL=debug` in the `jest` command in the `scripts` section of `package.json`
- setting the `--verbose` flag to `pnpm jest`
- some `dtrace` incantations to trigger the `bunyan` provider, but couldn't work around `dtrace` needing to run as `root` and not wanting to run `renovate` with elevated privilege
- setting `TERM=dumb` when running `pnpm jest`
- setting `--silent=false` both with and without `--verbose` when running `pnpm jest` (a bunch of stackoverflow discussion around whether one or the other or both are necessary, but those are all old, anyway)
- adding the following to the list of streams in the bunyan logger config in `lib/logger/index.ts`:
```js
    {
      name: 'debugfile',
      level: 'debug' as bunyan.LogLevel,
      path: '/Users/duvall/src/renovate/debug.log',
      type: 'raw',
    },
```

### Relevant debug logs

Not sure there are any.

**Comment by rarkins** [comment]
My hacky approach is to temporarily rename logger.x commands to console.warn, which are printed
